sdk/docs/core/CHANGELOG.md

```diff
@@ -4,8 +4,22 @@
 ## 1.0.0-preview.4 (Unreleased)
 
 
-## 1.0.0-preview.3 (Unreleased)
+## 1.0.0-preview.3 (2020-07-20)
 
+- Updated `az_result` values:
```

Comment:
Should we have something about 
```
- Default behavior for failed precondition changed to infinite loop instead of thread sleep
```

Reply:
Can you point me to the PR where that was changed?

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/866